feature/settings/src/main/kotlin/uk/govuk/app/settings/ui/SettingsScreen.kt

```diff
@@ -33,10 +53,98 @@ private fun SettingsScreen(
         onPageView()
     }
 
-    Column(modifier) {
-        Text("Settings Screen")
-        Button(onClick = onButtonClick) {
-            Text("Click me!")
+    Column {
+        BaseHeader(stringResource(R.string.screen_title))
+        Column(
+            modifier = Modifier.verticalScroll(rememberScrollState())
+        ) {
+            AboutTheApp(onButtonClick)
+            PrivacyAndLegal()
+            Spacer(Modifier.height(100.dp))
         }
     }
-}
\ No newline at end of file
+}
+
+@Composable
+private fun AboutTheApp(
+    onButtonClick: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    SettingsHeader(stringResource(R.string.about_title))
```

Comment:
We should either wrap the contents of the function in a column (and pass it the modifier) or mark the function as column scope (not sure what to do with the modifier in that case!)